src/transacting.md

```diff
@@ -281,9 +281,15 @@ curl -X POST "https://service.centrapay.com/payments/api/transactions.refund" \
 | 190        | 200       | {% break _ TRANSACTION_ALREADY_EXISTS %}          | A successful payment transaction already exists for a payment request.                                                                                                                                                                                    |
 | 191        | 500       | {% break _ OPTIMISTIC_LOCK_ERROR %}               | A resource was updated concurrently. Request should be retried after refreshing latest state if applicable.                                                                                                                                               |
+| 192        | 403       | {% break _ INSUFFICIENT_ASSET_BALANCE %}          | The asset has insufficient funds to pay the payment request                                                  |
+| 193        | 403       | {% break _ INVALID_MERCHANT_CONFIGURATION %}      | The merchant is not configured properly to satisfy the payment request, could be incorrect information or the merchant's credentials might be blocked by an external service                                                  | 
+| 194        | 403       | {% break _ INACTIVE_ASSET %}          | The asset has either expired or been blocked                                                  | 
 | 276        | 400       | {% break _ ALREADY_REFUNDED %}                    | The transaction has already been refunded                                                                                                                                                                                                                 |
 | 277        | 400       | {% break _ INVALID_AMOUNT %}                      | The refund requested is greater than the transaction amount                                                                                                                                                                                               |
-
+| 278        | 500       | {% break _ INTERNAL_SERVER_ERROR %}               | Something went wrong while trying to refund the request, we have received an error message and will figure out what went wrong.  |
+| 279        | 403       | {% break _ INVALID_TRANSACTION_TYPE %}            | The transaction attempted to be refunded is a refund which is not allowed                                                        |
+| 280        | 403       | {% break _ REPEAT_REFERENCE %}                    | A separate refund request for the same transaction has the same external reference, trying to refund the same transaction twice with the same external reference is not allowed. If the amount of the refund is the same we assume it is a repeat request and return 200 |
```

Comment:
The alignment on this table is a bit off on these rows and on the header. Could we make the |'s line up? <3